src/charts/donut/Donut.js

```diff
@@ -11,11 +11,21 @@ export default class Donut extends Component {
          */
         data: PropTypes.array,
 
+        /**
+         * Gets or Sets the centeredTextFunction of the chart. If function is provided the format will be changed by the custom function's value format. The default format function value is "${d.percentage}% ${d.name}". The callback will provide the data object with id, name, percentage, and quantity. Also provides the component added by the user in each data entry
+         */
+        centeredTextFunction: PropTypes.func,
+
         /**
          * Gets or Sets the colorSchema of the chart
          */
         colorSchema: PropTypes.arrayOf(PropTypes.string),
 
+        /**
+         * Gets or Sets the emptyDataConfig of the chart. If set and data is empty (quantity adds up to zero or there are no entries), the chart will render an empty slice with a given color (light gray by default)
+         */
+        emptyDataConfig: PropTypes.object,
```

Comment:
After `2.6.0` this should be switched to `emptySliceColor`: https://github.com/eventbrite/britecharts/pull/539